codespell_lib/data/dictionary.txt

```diff
@@ -7817,6 +7819,7 @@ cyclinders->cylinders
 cycular->circular
 cyle->cycle
 cylic->cyclic
+cylider->cylinder
```

Comment:
Can you add another row with the plural version too please.

Reply:
done.